samples/bluetooth/beacon/rv32m1_vega_ri5cy.overlay

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
can these overlay be in some common folder for the samples?

Reply:
I think I can do a couple of changes to the build/overlay system so the overlays for a platform are taken from some other directory (rather than the sample directory).
I'm thinking of having a per-platform/per SoC directory structure like for instance overlays/riscv/rv32m1/rv32m1_vega_ri5cy.overlay, which would get overlayed only if BT=y.
This PR is not the best container for such a change, as such I'd create a separate one that allows the overlays to be decoupled from the sample dir. Does this make se?

Reply:
this may be more a generic topic, I have no strong objection. May be @jhedberg has some comments?

Reply:
@galak, This was the overlays I was talking to you about re: DTS. Can you provide some guidance here?